test/cmd/legacy-script.sh

```diff
@@ -106,6 +106,12 @@ daemonsets="daemonsets"
 controllerrevisions="controllerrevisions"
 job="jobs"
 
+# Create a junit-style XML test report in this directory if set.
+KUBE_JUNIT_REPORT_DIR=${KUBE_JUNIT_REPORT_DIR:-}
```

Comment:
This line sets KUBE_JUNIT_REPORT_DIR from the same variable name with default empty string.
I could not find the reason why this line is necessary.

Reply:
You know, I think you're right.

I had copied that code block from here: https://github.com/kubernetes/kubernetes/blob/69c3e1934c0c2b01c53e765cb1f5ef8b4b95dfd6/hack/make-rules/test.sh#L80-L85

But taking a closer look at it, I agree.  It doesn't appear to do anything.

Reply:
Ah, OK. I missed the above code.
Thanks for updating.